### ISSUE TITLE
be-typescript-express quickstarter provisioning fails

### ISSUE THREAD

**Issue by gerardcl** [opendevstack/ods-quickstarters#120]
**Describe the bug**
be-typescript-express component provisioning fails when starting the pipeline. Kubernetes pod label too long (limit is 63).

**To Reproduce**
1. In ODS 2.x/master, go to provisioning app and provision be-typescript-express component
2. Jenkins provisioning fails

**Expected behavior**
 The component should be provisioned without issues

**Affected version (please complete the following information):**
 - OpenShift: 3.11
 - OpenDevStack: master, 2.x (might be any version)

**Log Output (ensure to remove any confidential information like tokens, project names, etc.**
```bash
[Pipeline] Start of Pipeline
[Pipeline] node
Running on Jenkins in /var/lib/jenkins/jobs/newproject-cd/jobs/newproject-cd-ods-quickstarter-be-typescript-express-be-typescript-express-2x/workspace
[Pipeline] {
[Pipeline] }
[Pipeline] // node
[Pipeline] podTemplate
[Pipeline] // podTemplate
[Pipeline] End of Pipeline
ERROR: Labels must follow required specs - https://kubernetes.io/docs/concepts/overview/working-with-objects/labels/#syntax-and-character-set: quickstarter-be-typescript-express-ods2test3-be-typescript-express
Finished: FAILURE
```

**Additional context**
We need to assure pod labels are no longer than 63 characters.

**Comment by clemensutschig** [comment]
this is all the same topics .. pick a short name and try again.. please ...

**Comment by gerardcl** [comment]
yes, that is the workaround I already implemented, but then there is another followup bug, will update the issue soon-ish

**Comment by borja44** [comment]
The component **jupyter-notebook** Quickstarter have the same issue

**Comment by michaelsauter** [comment]
I am moving this to `ods-provisioning-app`. The prov app should prevent names that are too long, otherwise feedback to the user is hard.

ALL quickstarters will have that issue, please do not open for every quickstarter.
